docs/SETUP.md

```diff
@@ -127,6 +127,13 @@ jmx:
 
 The security parameters can be updated during runtime and will automatically be picked up by ecc.
 
+If ecChrones is deployed in a multi-site environment where clients can't connect to Cassandra nodes in remote sites the remoteRouting must be disabled.
```

Comment:
A little explanation of what disabling remoteRouting entails (SERIAL consistency, clients only connecting to local cluster)